index.html

```diff
@@ -76,6 +72,105 @@ <h2 class="pdb-10">Question 2</h2>
               commodi nostrum, et cupiditate quisquam iure recusandae non autem
               alias? Fugit!
             </p>
+            <button
+              data-js="show-answer"
+              class="card__answer card__answer--pointer"
+            >
+              Show answer
+            </button>
+            <p data-js="hide-answer" class="center mgt-10 hide">
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Aut,
+              natus modi culpa necessitatibus doloribus nam deserunt enim
+              nostrum recusandae beatae earum eveniet eius corporis quia. Iusto
+              quasi totam aut? Ducimus?
+            </p>
+            <ul role="list" class="card__tags no-bul">
+              <li class="inline card__tags--style">tag 1</li>
+              <li class="inline card__tags--style">tag 2</li>
+              <li class="inline card__tags--style">tag 3</li>
+              <li class="inline card__tags--style">tag 4</li>
+            </ul>
+          </article>
+        </div>
+        <div data-js="create-content" class="hide">
+          <form
+            data-js="create"
+            class="create"
+            aria-labelledby="create-question-heading"
+            autocomplete="off"
+          >
+            <label class="create__label" for="input-question"
+              >Let us know your question:</label
+            >
+            <textarea
```

Comment:
You could create a div with the attribute data-js="create". Then you can address both textareas.

Reply:
Below in the JS-Files i saw that you have different allowed maxlenght for the textareas so forget what im writing about

Reply:
I am a little confused what you mean right now - maybe we could talk later on? 😊
actually the data-js="create" was not used at all, so I deleted it now 😅